src/hooks/network/useOnOfflineToExitSketchMode.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { renderHook } from '@testing-library/react'
+import { useOnOfflineToExitSketchMode } from '@src/hooks/network/useOnOfflineToExitSketchMode'
+import { vi } from 'vitest'
+import { ConnectionManager } from '@src/network/connectionManager'
+import { EngineConnectionEvents } from '@src/network/utils'
+
+describe('useOnOfflineToExitSketchMode.tsx', () => {
```

Comment:
Including the filename in the test names feels strange to me. I would change this to `useOnOfflineToExitSketchMode` since that is the function being tested. Ditto on the rest of these unit tests.